contrib/scoverage/api/src/ScoverageReportWorkerApi.scala

```diff
@@ -5,7 +5,12 @@ import mill.api.Ctx
 trait ScoverageReportWorkerApi {
   import ScoverageReportWorkerApi._
 
-  def report(reportType: ReportType, sources: Seq[os.Path], dataDirs: Seq[os.Path])(implicit
+  def report(
+      reportType: ReportType,
+      sources: Seq[os.Path],
+      dataDirs: Seq[os.Path],
+      sourceRoot: os.Path
+  )(implicit
```

Comment:
As the worker API is shared between all worker implementations, we need to keep/support the `sourceRoot` parameter, in case we decide to keep the Scoverage 1.x worker.